### ISSUE TITLE
[Bug]: Can't import data when create an index and cause service block.

### ISSUE THREAD

**Issue by chrysanthemum-boy** [infiniflow/infinity#914]
### Is there an existing issue for the same bug?

- [X] I have checked the existing issues.

### Branch name

main

### Commit ID

95562e9

### Actual behavior

Can't import data when create an index and cause service block.
```
    @pytest.mark.parametrize("check_data", [{"file_name": "enwiki_9.csv",
                                             "data_dir": common_values.TEST_TMP_DIR}], indirect=True)
    def test_with_fulltext_match(self, get_infinity_db, check_data):
        db_obj = get_infinity_db
        db_obj.drop_table("test_with_fulltext_match", ConflictType.Ignore)
        table_obj = db_obj.create_table("test_with_fulltext_match",
                                        {"doctitle": "varchar",
                                         "docdate": "varchar",
                                         "body": "varchar",})
                                         # "num": "int",
                                         # "vec": "vector, 4, float"
        table_obj.create_index("my_index",
                               [index.IndexInfo("body",
                                                index.IndexType.FullText,
                                                [index.InitParameter("ANALYZER", "segmentation")]),
                                ], ConflictType.Error)

        if not check_data:
            copy_data("enwiki_9.csv")
        test_csv_dir = common_values.TEST_TMP_DIR + "enwiki_9.csv"
        table_obj.import_data(test_csv_dir, {"delimiter": "\t"})
        res = table_obj.output(["*"]).to_pl()
        print(res)
```

### Steps to reproduce

```Markdown
Just run the function mentioned above.
```

**Comment by Ognimalf** [comment]
Fixed by #918

**Comment by chrysanthemum-boy** [comment]
It seems fixed.
